docs/integrations/destinations/snowflake.md

```diff
@@ -0,0 +1,109 @@
+# Snowflake
+
+## Overview
+
+The Airbyte Snowflake destination allows you to sync data to Snowflake.
+
+### Sync overview
+
+#### Output schema
+
+Each stream will be output into its own table in Snowflake. Each table will contain 3 columns:
+
+* `ab_id`: a uuid assigned by Airbyte to each event that is processed. The column type in Snowflake is `VARCHAR`.
+* `emitted_at`: a timestamp representing when the event was pulled from the data source. The column type in Snowflake is `TIMESTAMP WITH TIME ZONE`.
+* `data`: a json blob representing with the event data. The column type in Snowflake is `VARIANT`.
+
+#### Features
+
+This section should contain a table with the following format:
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+
+## Getting started
+
+We recommend creating an Airbyte-specific warehouse, database, schema, user, and role for writing data into Snowflake so 
+it is possible to track costs specifically related to Airbyte (including the cost of running this warehouse) and control permissions at a granular level. 
+Since the Airbyte user creates, drops, and alters tables, `OWNERSHIP` permissions are required in Snowflake. If you are not 
+following the recommended script below, please limit the `OWNERSHIP` permissions to only the necessary database and schema for the Airbyte user.
+
+We provide the following script to create these resources. 
+Before running, you must change the password to something secure. 
+You may change the names of the other resources if you desire.
+
+```
+-- set variables (these need to be uppercase)
+set airbyte_role = 'AIRBYTE_ROLE';
+set airbyte_username = 'AIRBYTE_USER';
+set airbyte_warehouse = 'AIRBYTE_WAREHOUSE';
+set airbyte_database = 'AIRBYTE_DATABASE';
+set airbyte_schema = 'AIRBYTE_SCHEMA';
+
+-- set user password
+set airbyte_password = 'password';
```

Comment:
is it chill for password to be password?

Reply:
oh i guess it gets revoked afterwards?

Reply:
Above it says to change this to a real password. Should I add something here as well?

Reply:
sorry. i just didn't see it. i might do `<INSERT-YOUR-PASSWORD` instead `password`